jbpm-flow/src/main/java/org/jbpm/workflow/instance/impl/WorkflowProcessInstanceImpl.java

```diff
@@ -751,6 +753,14 @@ public String getCorrelationKey() {
     public void setCorrelationKey(String correlationKey) {
         this.correlationKey = correlationKey;
     }
+    
+    public Date getStartDate() {
+        return startDate;
+    }
+    
+    public void setStartDate(Date startDate) {
+        this.startDate = startDate;
```

Comment:
here setter is fine though I would most likely rename it to internalSetStartDate so it's not directly seen as set* and in addition allow setting it only when it was not set yet.